### ISSUE TITLE
Submit of sidesheets hidden behind keyboard on mobile

### ISSUE THREAD

**Issue by gnunicorn** [acterglobal/a3#999]
You can access the submit buttons. I think we want to remodel the dialog to have a sticky bottom-bar with the buttons and if there isn't enough space for all the things, they need to be scrolled...

![Screenshot_1695720199](https://github.com/acterglobal/a3/assets/40496/72688570-67c6-4bd6-9003-c0f2dd920b3e)

**Comment by gtalha07** [comment]
Agree, we should use full screen bottom sheets for mobile with infinite scrolling for smaller screens.
